routes/routes.go

```diff
@@ -61,8 +62,8 @@ func OperatorRoutes(a *fiber.App, config *configs.ValkyrieConfig, client rest.HT
 	for _, c := range config.Providers {
 		operatorRouter, err := provider.OperatorFactory().
 			Build(c.Name, provider.OperatorArgs{
-				Config: c,
-				Client: client,
+				Config:     c,
+				HTTPClient: client,
```

Comment:
Agree that HTTPClient is a lot more clear. Perhaps rename argument too even

Reply:
Fixed!